reana_workflow_controller/app.py

```diff
@@ -42,6 +42,34 @@
 }
 
 
+@app.route('/workflows', methods=['GET'])
+def get_workflows():
+    """Get all workflows.
+    
```

Comment:
PEP8 expects here and empty line. Try to plug some linter to your editor so you notice this kind of errors before! :)